docs/0.3/upgrading_smart_contracts_for_administrators.md

```diff
@@ -33,14 +33,14 @@ and then deploying it.
 
```

Comment:
The note above refers to the Pike sc, should be Purchase Order

Reply:
updated